### PR TITLE
fixed axial blocks in ECAT pet

### PR DESCRIPTION
**Pull request by ludovicabrusaferri**
Fixed issue with number of axial blocks in z direction in ECAT PET

### CODE REVIEW

File: src/include/stir/IO/InputStreamFromROOTFileForECATPET.inl [hunk 1]
```diff
@@ -21,29 +21,29 @@ int
 InputStreamFromROOTFileForECATPET::
 get_num_rings() const
 {
-    return this->crystal_repeater_z;
+   return this->block_repeater_y * this->crystal_repeater_z;
```
Comment (KrisThielemans) [inline src/include/stir/IO/InputStreamFromROOTFileForECATPET.inl]: `block_repeater_z`?

Comment (ludovicabrusaferri) [reply src/include/stir/IO/InputStreamFromROOTFileForECATPET.inl]: Yes, i was about to fix it

Comment (ludovicabrusaferri) [reply src/include/stir/IO/InputStreamFromROOTFileForECATPET.inl]: I think there is an example in https://github.com/UCL/STIR/blob/9be73d4615e4cd53a6fbc2abdd6ebf4e8f85e59d/src/include/stir/IO/InputStreamFromROOTFileForECATPET.h#L51-L66

i added the keyword and also corrected the current crystal keywords that were wrong.

**Comment by KrisThielemans** [comment]
I meant like https://github.com/UCL/STIR/blob/e2f0bcb74af6bbf067785e07f58a8430faf3c64d/examples/samples/root_header.hroot. could you create one like that as well? let's call it `root_header_ECAT.hroot` and add a comment in there that this is for an ECAT GATE system

File: examples/samples/root_headerECAT.hroot [hunk 2]
```diff
@@ -0,0 +1,61 @@
+ROOT header := 
+
```
Comment (KrisThielemans) [inline examples/samples/root_headerECAT.hroot]: add a comment like `This is an example file for a ROOT file generated with GATE for a Siemens mMR-like scanner (but using an 8x9 block) and using the ECAT system as output.`

**Pull request merged**